### PR TITLE
feat(globalDecision): show warning if the candidate license is added

### PR DESCRIPTION
**Pull request by shaheemazmalmmd**
## Description

* check if the candidate license is added to the list 
* if added the show message

## How to test

* Upload a package X.
* Create a candidate license.
* Add the candidate license to license table in single file view through user decision.
* Check "Apply decision to all future occurrences of this file" option.

### CODE REVIEW

File: src/lib/php/Dao/ClearingDao.php [hunk 1]
```diff
@@ -1022,4 +1022,24 @@ public function getTotalDecisionCount($uploadId, $groupId)
     $foundCounter = $this->dbManager->getSingleRow($sql, $params, $statement);
     return $foundCounter['cnt'];
   }
+
+  /**
+   * @param uploadTreeId
+   * @return count
+   */
+  public function getCandidateCountForItemId($uploadTreeId)
```
Comment (GMishx) [inline src/lib/php/Dao/ClearingDao.php]: Can you please add this check to `createDecisionFromEvents()` as well so we can make sure the check happens at backend as well?

https://github.com/fossology/fossology/blob/b5a389be8ef413fc488b56700ea23bdf3ff9c73a/src/lib/php/Dao/ClearingDao.php#L334

Comment (GMishx) [reply src/lib/php/Dao/ClearingDao.php]: Also for `markDirectoryAsDecisionType()`

https://github.com/fossology/fossology/blob/b5a389be8ef413fc488b56700ea23bdf3ff9c73a/src/lib/php/Dao/ClearingDao.php#L753

File: src/lib/php/Dao/ClearingDao.php [hunk 2]
```diff
@@ -333,6 +333,10 @@ public function removeWipClearingDecision($uploadTreeId, $groupId)
    */
   public function createDecisionFromEvents($uploadTreeId, $userId, $groupId, $decType, $scope, $eventIds)
   {
+    if ( ($scope == DecisionScopes::REPO) &&
+         !empty($this->clearingDao->getCandidateCountForItemId($itemId))) {
```
Comment (GMishx) [inline src/lib/php/Dao/ClearingDao.php:337]:
```suggestion
         !empty($this->getCandidateCountForItemId($uploadTreeId))) {
```

Comment (shaheemazmalmmd) [reply src/lib/php/Dao/ClearingDao.php:337]: Oops. Missed it copy .

**Comment by GMishx** [comment]
@shaheemazmalmmd , can you please modify the commit message so that we don't get the weird `...` at the end?

**Pull request merged**